shell/pages/c/_cluster/fleet/index.vue

```diff
@@ -284,223 +425,510 @@ export default {
+                tabindex="0"
+                :aria-label="t(`${ isWorkspaceCollapsed[workspace.id] ? 'expand': 'collapse' }-${ workspace.id }`)"
+                :class="{
+                  ['icon icon-lg icon-chevron-right']: isWorkspaceCollapsed[workspace.id],
+                  ['icon icon-lg icon-chevron-down']: !isWorkspaceCollapsed[workspace.id],
+                }"
+                @click="toggleCard(workspace.id)"
+                @keydown.space.enter.stop.prevent="toggleCard(workspace.id)"
               />
-            </template>
-
-            <template #cell:target="{row}">
-              {{ row.targetInfo.modeDisplay }}
-            </template>
-          </ResourceTable>
-        </template>
-      </CollapsibleCard>
+            </div>
+          </div>
+        </div>
+        <div
+          v-if="!isWorkspaceCollapsed[workspace.id]"
+          class="card-panel-expand mt-10"
+          :data-testid="`fleet-dashboard-expanded-panel-${ workspace.id }`"
+        >
+          <div class="actions">
+            <div
+              v-if="false"
+              class="type-filters"
+            >
+              <Checkbox
+                :data-testid="'fleet-dashboard-filter-git-repos'"
+                :value="typeFilter[workspace.id]?.[FLEET.GIT_REPO]"
+                @update:value="selectType(workspace.id, FLEET.GIT_REPO, $event)"
+              >
+                <template #label>
+                  <i class="icon icon-lg icon-git mr-5" />
+                  <span class="label">{{ t('fleet.dashboard.cards.filters.gitRepos') }}</span>
+                </template>
+              </Checkbox>
+              <Checkbox
+                :data-testid="'fleet-dashboard-filter-helm-ops'"
+                :value="typeFilter[workspace.id]?.[FLEET.HELM_OP]"
+                @update:value="selectType(workspace.id, FLEET.HELM_OP, $event)"
+              >
+                <template #label>
+                  <i class="icon icon-lg icon-helm mr-5" />
+                  <span class="label">{{ t('fleet.dashboard.cards.filters.helmOps') }}</span>
+                </template>
+              </Checkbox>
+            </div>
+            <div
+              v-if="viewMode === 'flat'"
+              class="create-button"
+            >
+              <router-link
+                :to="createRoute"
+                class="btn role-primary"
+              >
+                {{ t('fleet.gitRepo.intro.add') }}
+              </router-link>
+            </div>
+          </div>
+          <div
+            v-if="viewMode === 'cards'"
+            class="cards-panel"
+          >
+            <div
+              v-for="(state, j) in applicationStates[workspace.id]"
+              :key="j"
+              :data-testid="`state-panel-${ state.stateDisplay }`"
+            >
+              <div
+                v-if="cardResources[workspace.id][state.stateDisplay]?.length"
+                class="card-panel"
+              >
+                <div
```

Comment:
The focus ring has a white border in dark mode

![image](https://github.com/user-attachments/assets/0ba0eeb2-3ec6-46f0-87b4-6186a08185cb)